conftest.py

```diff
@@ -1,29 +1,36 @@
+"""PyTest Fixtures."""
 import os
+from test import RunFromText
 
 import pytest
 
-from ansiblelint.rules import RulesCollection
+import ansiblelint.rules as rules
 
-from . import RunFromText
+
+def pytest_configure(config):
+    """Add rule folder to test include for running embedded rule tests."""
+    config.args.append(os.path.dirname(rules.__file__))
```

Comment:
When you run `pytest --help`, it explodes with
```python-traceback
  File "test/conftest.py", line 12, in pytest_configure
    config.args.append(os.path.dirname(rules.__file__))
AttributeError: 'Config' object has no attribute 'args'
```

Reply:
Use `--pyargs` to avoid such problems:
https://github.com/ansible/ansible-lint/pull/1009/files#diff-513c9b6861fc981fb2c708db477f542cR17-R61. This will also make the setting more declarative.